op-node/flags/flags.go

```diff
@@ -65,6 +65,13 @@ var (
 		Destination: new(string),
 		Category:    RollupCategory,
 	}
+	BuilderAPIAddr = &cli.StringFlag{
```

Comment:
this can be extended to an array for multiple builders?

Reply:
I think that changes expected behavior though because we assume only one builder responds ATM